<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="/assets/style.css"/>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700"/>
    <script src="/assets/fontawesome/attribution.js"></script>
    <link rel="icon" href="https://www.k-state.edu/favicon.ico" type="image/x-icon">
    <title>Blog | Liu Lab</title>
    <!-- metadata for SEO -->
    <meta property="og:title" content="Blog | Liu Lab"/>
    <link rel="canonical" href="PlantGGHLab.github.io/blog.html"/>
    <meta property="og:url" content="PlantGGHLab.github.io/blog.html"/>
    <meta property="og:site_name" content="Liu Lab"/>
    <meta property="og:locale" content="en_US"/>
</head>
<body><header role="banner">
  <a class="ksu-wordmark" href="https://k-state.edu"></a>
  <nav>
      <ul>
		  <li><a class="site-title" rel="author" href="/">Liu Lab</a></li>
		  
			  <li>
				  
					  <a href="/people">People</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/projects">Projects</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/publications">Publications</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/teaching">Teaching</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/blog">Blog</a>
				  
			  </li>
		  
      </ul>
  </nav>
</header>
<div class="wrapper">
			<aside class="left-sidebar"><h3>Links</h3>
<ul class="links">
    <li>
        <svg class="fa-icon">
            <use href="/assets/fontawesome/sprites/brands.svg#github"/>
        </svg>
        <a href="https://github.com/PlantGGHLab">
            PlantGGHLab
        </a>
    </li>
    <li>
        <svg class="fa-icon">
            <use href="/assets/fontawesome/sprites/brands.svg#twitter"/>
        </svg>
        <a href="https://twitter.com/liu3zhen">
            liu3zhen
        </a>
    </li>
</ul>

<hr/>

<address style="white-space: pre; font-style: normal">
    Liu Lab
    4022 Throckmorton Hall
    1712 Claflin Rd
    Manhattan, KS 66502
    <a href="tel:+17855321412">+1 (785) 532-1412 </a>
</address>
</aside>
			<main class="page-content" aria-label="Content">
				<h1>Blog</h1>
				
<details open="">
  <summary>
    <h2>2020</h2>
  </summary>

  <h3 id="jan-11">Jan 11</h3>

  <p><a href="assets/images/pag-2020.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/pag-2020.jpg" class="blog-img" />
</a></p>

  <p>Dr. Cheng He and graduate students Guifang Lin and Ha Le presented their research at PAG (International Plant &amp; Animal
Genome) conference.</p>

</details>

<hr />

<details>
  <summary>
    <h2>2019</h2>
  </summary>

  <h3 id="dec-30">Dec 30</h3>

  <p><a href="assets/images/labgroup-2019.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/labgroup-2019.jpg" class="blog-img" />
</a></p>

  <p>Group photo 2019</p>

  <h3 id="jul-13">Jul 13</h3>

  <p><a href="assets/images/summer-party-2019.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/summer-party-2019.jpg" class="blog-img" />
</a></p>

  <p>BBQ party at Tuttle Creek State Park</p>

  <h3 id="jun-15">Jun 15</h3>

  <p>Dr. Huakun Zheng joined our lab as a visiting scholar. He will be working on
genomic dynamics of fungal mini-chromosomes.</p>

  <h3 id="apr-26">Apr 26</h3>

  <p><a href="assets/images/field-2019.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/field-2019.jpg" class="blog-img" />
</a></p>

  <p>Field planting team</p>

  <h3 id="jan-12">Jan 12</h3>

  <p>Welcome Dr. Bhanu Kalia to join the lab! She will work on the wheat leaf and
stripe rust projects.</p>

</details>

<hr />

<details>
  <summary>
    <h2>2018</h2>
  </summary>

  <h3 id="dec-30-1">Dec 30</h3>

  <p>Welcome Dr. Mingxia Zhao to join us starting from Dec 28th 2019! She will work on our NSF-funded project to understand maize regeneration.</p>

  <h3 id="oct-31">Oct 31</h3>

  <p>Two maize glossy genes were identified through the collaboration among Wang lab at CAAS, Schnable lab at ISU, and our lab. Now the manuscript was published in the Plant journal.</p>

  <h3 id="sep-01">Sep 01</h3>

  <p>Excited to start the NSF PGRP project. Will work closely with Sunghun Park (K-State), Frank White (UF), Myeong-Je Cho (UC-Berkeley), and Hairong Wei (MTU). The project was briefly described in <a href="https://www.k-state.edu/today/announcement.php?id=43475"> K-State news</a>.</p>

  <h3 id="aug-24">Aug 24</h3>

  <p><a href="assets/images/labgroup-2018.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/labgroup-2018.jpg" class="blog-img" />
</a></p>

  <p>Group photo 2018</p>

  <h3 id="aug-23">Aug 23</h3>

  <p>Welcome Marlenny Grullon, an undergraduate student majoring in Agronomy, to join the lab.</p>

  <h3 id="aug-12">Aug 12</h3>

  <p>Welcome Jinguang Lin, a PhD student in Statistics, to join the lab.</p>

  <h3 id="jul-15">Jul 15</h3>

  <p>Thrilled to have our Goss’s wilt project funded by NSF/NIFA Plant Biotic Interactions Program. Will work with Frank White at UF and Alison Robertson at ISU on this exciting project.</p>

  <h3 id="apr-28">Apr 28</h3>

  <p><a href="assets/images/field-2018.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/field-2018.jpg" class="blog-img" />
</a></p>

  <p>First planting day in 2018.</p>

  <h3 id="feb-10">Feb 10</h3>

  <p>Our XP-CNV paper is officially online. A general approach was developed to associate CNV with phenotypic traits. Thank all co-authors for their contributions.</p>

</details>

<hr />

<details>
  <summary>
    <h2>2017</h2>
  </summary>

  <h3 id="dec-16">Dec 16</h3>

  <p>Welcome Dr. Cheng He to join the lab. Dr. He will work on gene/isoform transcriptomics of maize.</p>

  <h3 id="sep-18">Sep 18</h3>

  <p>Our <a href="https://academic.oup.com/nar/article/doi/10.1093/nar/gkx853/4210942/tGBS-genotypingbysequencing-enables-reliable">tGBS paper</a> was accepted by Nucleic Acids Research. Congratulations to all authors!</p>

  <h3 id="jun-04">Jun 04</h3>

  <p>Northeast Forestry University visiting in China hosted by Drs. Guifeng Liu, Hairong Wei, Yanting Jiang (6/4-6). Very impressed by tree genetic studies there.</p>

  <h3 id="may-24">May 24</h3>

  <p>Fujian Agriculture and Forestry University visiting in China (5/24-25). Discussions with Dr. Weiren Wu for a potential collaboration and student visiting to KSU.</p>

  <h3 id="may-04">May 04</h3>

  <p>Sanzhen’s advisor, Dr. Patrick Schnable, visited KSU and had a nice discussion with lab members.</p>

  <h3 id="apr-28-1">Apr 28</h3>

  <p><a href="assets/images/field-2017.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/field-2017.jpg" class="blog-img" />
</a></p>

  <p>Great field team</p>

  <h3 id="apr-18">Apr 18</h3>

  <p>Sanzhen’s Colorado State University visiting hosted by Dr. Kirk Broders (4/18-20). Great visit and had nice interactions with postdocs and graduate students</p>

  <h3 id="apr-16">Apr 16</h3>

  <p><a href="assets/images/labgroup-2017.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/labgroup-2017.jpg" class="blog-img" />
</a></p>

  <p>Group photo 2017</p>

  <h3 id="mar-13">Mar 13</h3>

  <p>Congratulations to Pierre Migeon for his successful thesis defense!</p>

  <h3 id="feb-10-1">Feb 10</h3>

  <p>Our k-mer analysis for comparative genomics was accepted by Scientific Report and is online today. <a href="http://www.nature.com/articles/srep42444">Click here</a> for the manuscript.</p>

  <h3 id="feb-08">Feb 08</h3>

  <p>Dr. Hairong Wei’s visiting. He gave a great talk for summarizing the algorithms that he and his team developed for constructing gene regulation networks. We also found our collaboration projects between the two labs.</p>

  <h3 id="jan-17">Jan 17</h3>

  <p>Guifang Lin starts her Ph.D. study this semester. She will work on the genetic bases of maize regeneration and wheat resistance to leaf rust.</p>

</details>

<hr />

<details>
  <summary>
    <h2>2016</h2>
  </summary>

  <h3 id="dec-29">Dec 29</h3>

  <p>Dr. Junjie Fu started his 40 days short visiting.</p>

  <h3 id="nov-03">Nov 03</h3>

  <p>Dr. James Schnable’s visiting and discussion for the collaboration.</p>

  <h3 id="oct-13">Oct 13</h3>

  <p>Dr. Jun Zheng’s short visiting. Dr. Zheng is an associate professor from CAAS.</p>

  <h3 id="oct-08">Oct 08</h3>

  <p>Welcome Daniel Park to become our Bioinformatician</p>

  <h3 id="sep-22">Sep 22</h3>

  <p>Dr. Erliang Zeng’s visiting to the lab. Dr. Zeng is an assistant professor at University of South Dakota. He gave a nice talk in our department about his network studies.</p>

  <h3 id="may-25">May 25</h3>

  <p><a href="assets/images/flood-2016.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/flood-2016.jpg" class="blog-img" />
</a></p>

  <p>Summer nursery suffered a severe flood.</p>

  <h3 id="may-08">May 08</h3>

  <p>Welcome Dan Park to join the lab for his 3-month Summer internship</p>

  <h3 id="apr-22">Apr 22</h3>

  <p>summer nursery planting</p>

  <h3 id="mar-30">Mar 30</h3>

  <p><a href="assets/images/labgroup-2016.jpg" target="_blank" rel="noopener noreferrer" class="blog-img">
    <img src="assets/images/labgroup-2016.jpg" class="blog-img" />
</a></p>

  <p>Group photo 2016</p>

  <h3 id="mar-02">Mar 02</h3>

  <p>Dr. Yihui Xie’s visiting to the Department of Plant Pathology. He introduced multiple useful tools (e.g., Bookdown) to us and answered many questions related to R programming.</p>

  <h3 id="jan-10">Jan 10</h3>

  <p>Welcome our new postdoc Dr. Ying Hu to join us starting from Jan 10th 2016!</p>

</details>

<hr />

<details>
  <summary>
    <h2>2015</h2>
  </summary>

  <h3 id="dec-21">Dec 21</h3>

  <p>Zhao Peng’s work “Long read and single molecule DNA sequencing simplifies genome assembly and TAL effector gene analysis of Xanthomonas translucens” was accepted at BMC Genomics. Congratulations Zhao!</p>

</details>


			</main>
			<aside class="right-sidebar">
				<div class="news">
					<h3 id="news">News</h3>

<ul>
  <li>The KAD manuscript was accepted <a href="https://academic.oup.com/nargab">NAR GAB</a>,
which uses k-mer profiling to effectively identify and locate base errors of
genome assemblies. The KAD package can be found on
<a href="https://github.com/liu3zhenlab/KAD">Github</a>. Use &amp; feedback is appreciated.</li>
  <li>Guifang Lin passed her preliminary exam on 4/8/2020. This is an important step toward her PhD. Congratulations!</li>
  <li>We identified and verified two maize glossy genes with the information of gene
co-expression networks. The manuscript was accepted by the Plant Journal. <a href="https://doi.org/10.1111/tpj.14140">Read more</a></li>
  <li>We are honored to receive the NSF PGRP award to work on maize transformation. <a href="https://www.k-state.edu/today/announcement.php?id=43475">Read more</a></li>
  <li>New manuscript with Schnable labs in Molecular Biology and Evolution: Intragenic
Meiotic Crossovers Generate Novel Alleles with Transgressive Expression Levels. <a href="https://doi.org/10.1093/molbev/msy174">Read more</a></li>
  <li>Exciting News: NIFA is funding, through NSF/NIFA Plant Biotic Interactions
Program, our 3-year project to genetically dissect bacterial virulence and host
resistance of maize Goss’s wilt. <a href="https://www.k-state.edu/today/announcement/?id=43466">Read more</a></li>
  <li>Our latest publication using XP-CNV to discover disease associated genes:
Analysis of Extreme Phenotype Bulk Copy Number Variation (XP-CNV) Identified the
Association of rp1 with Resistance to Goss’s Wilt of Maize <a href="https://doi.org/10.3389/fpls.2018.00110">Read more</a></li>
  <li>A new genotyping solution: tGBS® genotyping-by-sequencing enables reliable
genotyping of heterozygous loci <a href="https://doi.org/10.1093/nar/gkx853">Read more</a></li>
  <li>Our K-mer study to characterize genomes: A K-mer method for comparative genomics
and genetic mapping of copy number variation: Unbiased K-mer Analysis Reveals
Changes in Copy Number of Highly Repetitive Sequences During Maize Domestication
and Improvement <a href="https://doi.org/10.1038/srep42444">Read more</a></li>
  <li>Congratulations to Dr. Yanhai Yin who led the Rd26 work: RD26 mediates crosstalk
between drought and brassinosteroid signalling pathway <a href="https://doi.org/10.1038/ncomms14573">Read more</a></li>
</ul>

<p><a href="./news.md">Older news</a></p>
					
				</div>
			</aside>	  
		</div>
		
	</body>
</html>
<!--
TODOs
- compute the widths in style.scss based on some width threshold of wide and wider monitor.
-->
