<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="/assets/style.css"/>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700"/>
    <script src="/assets/fontawesome/attribution.js"></script>
    <link rel="icon" href="https://www.k-state.edu/favicon.ico" type="image/x-icon">
    <title>Projects | Liu Lab</title>
    <!-- metadata for SEO -->
    <meta property="og:title" content="Projects | Liu Lab"/>
    <link rel="canonical" href="PlantGGHLab.github.io/projects.html"/>
    <meta property="og:url" content="PlantGGHLab.github.io/projects.html"/>
    <meta property="og:site_name" content="Liu Lab"/>
    <meta property="og:locale" content="en_US"/>
</head>
<body><header role="banner">
  <a class="ksu-wordmark" href="https://k-state.edu"></a>
  <nav>
      <ul>
		  <li><a class="site-title" rel="author" href="/">Liu Lab</a></li>
		  
			  <li>
				  
					  <a href="/people">People</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/projects">Projects</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/publications">Publications</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/teaching">Teaching</a>
				  
			  </li>
		  
			  <li>
				  
					  <a href="/blog">Blog</a>
				  
			  </li>
		  
      </ul>
  </nav>
</header>
<div class="wrapper">
			<aside class="left-sidebar"><h3>Links</h3>
<ul class="links">
    <li>
        <svg class="fa-icon">
            <use href="/assets/fontawesome/sprites/brands.svg#github"/>
        </svg>
        <a href="https://github.com/PlantGGHLab">
            PlantGGHLab
        </a>
    </li>
    <li>
        <svg class="fa-icon">
            <use href="/assets/fontawesome/sprites/brands.svg#twitter"/>
        </svg>
        <a href="https://twitter.com/liu3zhen">
            liu3zhen
        </a>
    </li>
</ul>

<hr/>

<address style="white-space: pre; font-style: normal">
    Liu Lab
    4022 Throckmorton Hall
    1712 Claflin Rd
    Manhattan, KS 66502
    <a href="tel:+17855321412">+1 (785) 532-1412 </a>
</address>
</aside>
			<main class="page-content" aria-label="Content">
				<h1>Projects</h1>
				<h1 id="maize-embryogenesis-and-regeneration">Maize Embryogenesis and Regeneration</h1>
<p>CRISPRs provide great potential for the elucidation of maize gene functions and
trait improvements. However, current genetic and genomic resources as well as
transformation capacity in maize are insufficient for the full utilization of
genome engineering tools. We are generating public resources that include a
high-quality genome assembly of a highly transformation amenable maize inbred
line A188 and double haploid (DH) lines from a cross of A188 with B73, a
transformation recalcitrant inbred variety and the maize reference genome. We
are genetically mapping the genetic elements controlling embryogenesis and
regeneration. Our goal is to identify genes through map-based cloning with the
assistance from multilayered gene regulatory networks, and apply the genes to
enhance maize transformation.</p>

<h1 id="maize-diseases---gosss-wilts">Maize Diseases - Goss’s Wilts</h1>
<p>Goss’s wilt is a maize disease, caused by Clavibacter michiganensis subsp.
nebraskensis (CMN) that was first identified in 1969. The disease has not been a
serious problem until recent re-emergence in the Great Plains region. While a
number of Clavibacter-related diseases have been characterized, few genetic
resources have been identified specifically for Goss’s wilt. We performed
traditional GWAS, traditional QTL, XP-GWAS, XP-CNV to dissect disease
resistance. Meanwhile, we performed genome sequencing for multiple Cmn strains.
Our project goal is to provide extensive genetic information into plant
resistance and bacterial virulence with regard to the re-emergence of Goss’s
Wilt disease of maize. Our ultimate goal is to identify resistance genes in
plant host and virulence elements in bacteria.</p>

<h1 id="genomic-structural-variation">Genomic Structural Variation</h1>
<p>Genomes contain structural variations (SV) that are comprised of unbalanced
forms (copy number variation) and balanced forms (sequences in same copy number
but at different genomic locations), and SV has been implicated in phenotype
variation. However, SV often occurs on a scale that is difficult to be
characterized with a single approach. We are attempting to combine multiple
approaches, next-generation sequencing, genetic approach, and BioNano genome
mapping technology to characterize SV.</p>

<h1 id="change-in-gene-expression-in-maize-under-abiotic-conditions">Change in Gene Expression in Maize Under Abiotic Conditions</h1>
<p>The genome-wide messenger RNA expression (mRNA), including splicing events, and
small RNA (sRNA) expression are tractable at the unprecedented level with the
utility of the next-generation sequencing technology. This project is designed
to measure mRNA and sRNA from plant individuals under different controlled
abiotic stresses. Through quantifying the change in the expression of genes,
splicing sites, and sRNAs in response to abiotic conditions, we hope to find
some candidates as the targets that can be used to enhance plant tolerance to
extremely environmental conditions. sRNA and mRNA co-expression networks were
also constructed to understand the change in expression correlation under
drought condition.</p>

<h1 id="wheat-diseases---leaf-rust-and-yellow-rust">Wheat Diseases - Leaf Rust and Yellow Rust</h1>
<p>Gene mapping is a process to find trait-associated genetic markers. The gene
mapping process is an important step towards identifying genes responsible for
phenotypic traits. In wheat, identification of effective (or polymorphic)
genetic markers and gene mapping are tedious. The BSR-Seq analysis, combining
RNA-Seq and bulked segregant analysis, was developed and has been successfully
used to facilitate mapping and cloning many genes in species other than wheat.
This project is to adapt this method to the wheat system. Currently, we are
applying BSR-Seq to map several genes in wheat, including leaf rust gene Lr42
and several unknown yellow rust genes in a diploid wild relative of wheat.</p>

			</main>
			<aside class="right-sidebar">
				<div class="news">
					<h3 id="news">News</h3>

<ul>
  <li>The KAD manuscript was accepted <a href="https://academic.oup.com/nargab">NAR GAB</a>,
which uses k-mer profiling to effectively identify and locate base errors of
genome assemblies. The KAD package can be found on
<a href="https://github.com/liu3zhenlab/KAD">Github</a>. Use &amp; feedback is appreciated.</li>
  <li>Guifang Lin passed her preliminary exam on 4/8/2020. This is an important step toward her PhD. Congratulations!</li>
  <li>We identified and verified two maize glossy genes with the information of gene
co-expression networks. The manuscript was accepted by the Plant Journal. <a href="https://doi.org/10.1111/tpj.14140">Read more</a></li>
  <li>We are honored to receive the NSF PGRP award to work on maize transformation. <a href="https://www.k-state.edu/today/announcement.php?id=43475">Read more</a></li>
  <li>New manuscript with Schnable labs in Molecular Biology and Evolution: Intragenic
Meiotic Crossovers Generate Novel Alleles with Transgressive Expression Levels. <a href="https://doi.org/10.1093/molbev/msy174">Read more</a></li>
  <li>Exciting News: NIFA is funding, through NSF/NIFA Plant Biotic Interactions
Program, our 3-year project to genetically dissect bacterial virulence and host
resistance of maize Goss’s wilt. <a href="https://www.k-state.edu/today/announcement/?id=43466">Read more</a></li>
  <li>Our latest publication using XP-CNV to discover disease associated genes:
Analysis of Extreme Phenotype Bulk Copy Number Variation (XP-CNV) Identified the
Association of rp1 with Resistance to Goss’s Wilt of Maize <a href="https://doi.org/10.3389/fpls.2018.00110">Read more</a></li>
  <li>A new genotyping solution: tGBS® genotyping-by-sequencing enables reliable
genotyping of heterozygous loci <a href="https://doi.org/10.1093/nar/gkx853">Read more</a></li>
  <li>Our K-mer study to characterize genomes: A K-mer method for comparative genomics
and genetic mapping of copy number variation: Unbiased K-mer Analysis Reveals
Changes in Copy Number of Highly Repetitive Sequences During Maize Domestication
and Improvement <a href="https://doi.org/10.1038/srep42444">Read more</a></li>
  <li>Congratulations to Dr. Yanhai Yin who led the Rd26 work: RD26 mediates crosstalk
between drought and brassinosteroid signalling pathway <a href="https://doi.org/10.1038/ncomms14573">Read more</a></li>
</ul>

<p><a href="./news.md">Older news</a></p>
					
				</div>
			</aside>	  
		</div>
		
	</body>
</html>
<!--
TODOs
- compute the widths in style.scss based on some width threshold of wide and wider monitor.
-->
